consensus/types/src/subnet_id.rs

```diff
@@ -77,48 +76,16 @@ impl SubnetId {
     /// Computes the set of subnets the node should be subscribed to during the current epoch,
     /// along with the first epoch in which these subscriptions are no longer valid.
     #[allow(clippy::arithmetic_side_effects)]
-    pub fn compute_subnets_for_epoch<E: EthSpec>(
+    pub fn compute_attestation_subnets(
```

Comment:
The comment above needs to be updated as we no longer pass epoch to this function.

Reply:
Yep, done